### PR TITLE
{data}[foss/2019a] dask 2.3.0 and bokeh 1.3.4 for Python 3.7.2

### PR DESCRIPTION
**Pull request by branfosj**

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/074dccce5e9c72f9294712f9ed5f1713 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b936fca9b0aed4e5ae1cdbfa3fe96cc0 for a full test report.

File: easybuild/easyconfigs/b/bokeh/bokeh-1.3.4-foss-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }),
+    ('Jinja2', '2.10.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/J/Jinja2'],
+        'checksums': ['065c4f02ebe7f7cf559e49ee5a95fb800a9e4528727aec6f24402a5374c65013'],
+    }),
+    ('tornado', '6.0.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/tornado'],
+        'checksums': ['c845db36ba616912074c5b1ee897f8e0124df269468f25e4fe21fe72f6edd7a9'],
+    }),
+    ('packaging', '19.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/b/bokeh/bokeh-1.3.4-foss-2019a-Python-3.7.2.eb]: @branfosj `MarkupSafe`, `Jinja2` and `packaging` are already included with Python 3.7.2, so you can drop these from `exts_list` (you'll still need `tornado` + `bokeh` itself)

File: easybuild/easyconfigs/d/dask/dask-2.3.0-foss-2019a-Python-3.7.2.eb [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    ('PyYAML', '5.1'),
+    ('SciPy-bundle', '2019.03'),
+    ('bokeh', '1.3.4', versionsuffix),
+]
+
+use_pip = True
+
+exts_list = [
+    ('fsspec', '0.4.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/fsspec'],
```
Comment (boegel) [inline easybuild/easyconfigs/d/dask/dask-2.3.0-foss-2019a-Python-3.7.2.eb]: @branfosj You can avoid listing `source_urls` for every extension by using the following:

```python
exts_default_options = {'source_urls': [PYPY_SOURCE]}

exts_list = [
    ('fsspec', '0.4.1', {
        'checksums': ['e37d78590e7a9adb5b165c9c5d1b650f08017173b40eb1efbdc02d6bdb0c753b'],
    ),
    ...
```

File: easybuild/easyconfigs/d/dask/dask-2.3.0-foss-2019a-Python-3.7.2.eb [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        'checksums': ['b58a3ece521cf541bf90b81f89e9ed478d7cdea4bbe70b56a6cfedbd3b360209'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/dask-%s' % x for x in ['mpi', 'remote', 'scheduler', 'ssh', 'submit', 'worker']],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```
Comment (boegel) [inline easybuild/easyconfigs/d/dask/dask-2.3.0-foss-2019a-Python-3.7.2.eb]: You can drop this one when using `PythonBundle`

Comment (branfosj) [reply easybuild/easyconfigs/d/dask/dask-2.3.0-foss-2019a-Python-3.7.2.eb]: done

File: easybuild/easyconfigs/d/dask/dask-2.3.0-foss-2019a-Python-3.7.2.eb (hunk 2 above)
Comment (branfosj) [reply easybuild/easyconfigs/d/dask/dask-2.3.0-foss-2019a-Python-3.7.2.eb]: done

File: easybuild/easyconfigs/b/bokeh/bokeh-1.3.4-foss-2019a-Python-3.7.2.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/b/bokeh/bokeh-1.3.4-foss-2019a-Python-3.7.2.eb]: done

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/82ad276a26a91aeb2699221fc47d1c92 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0211u12a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d9be4515a7fa94e4bb5d252f15a5746e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3105.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/eeb4f314a29ba7537d2c7138f6983a2a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2661.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/dceab808a0f839748c5e3d18f9d50bb1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2113.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/500d20e6ad537270f8b99ab0211e554c for a full test report.

**Pull request merged**